index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/158414823-4173f6aa-c844-45eb-ac2d-c87865bea573.png)

Therefore:

- [ ] Please, kindly add CSS styles to match the wireframe presented on the project requirements.

That way, your project will look beautiful! 💪